torchci/pages/metrics.tsx

```diff
@@ -300,6 +300,125 @@ export function TimeRangePicker({
   );
 }
 
+/**
+ * Allows the user to pick the TTS metrics.
+ */
+export function TtsPercentilePicker({
+  ttsPercentile,
+  setTtsPercentile,
+}: {
+  ttsPercentile: number;
+  setTtsPercentile: any;
+}) {
+  function handleChange(e: SelectChangeEvent<number>) {
+    setTtsPercentile(e.target.value as number);
+  }
+
+  return (
+    <>
+      <FormControl>
+        <InputLabel id="tts-percentile-picker-select-label">Percentile</InputLabel>
+        <Select
+          defaultValue={0.50}
+          label="Percentile"
+          labelId="tts-percentile-picker-select-label"
+          onChange={handleChange}
+        >
+          <MenuItem value={-1.0}>avg</MenuItem>
+          <MenuItem value={0.50}>p50</MenuItem>
+          <MenuItem value={0.90}>p90</MenuItem>
+          <MenuItem value={0.95}>p95</MenuItem>
+          <MenuItem value={0.99}>p99</MenuItem>
+          <MenuItem value={1.00}>p100</MenuItem>
```

Comment:
p100 is just max?

Reply:
Yeah, it shows the outliers, which might be useful in certain cases